cfme/middleware/server.py

```diff
@@ -275,6 +280,32 @@ def is_stopped(self):
         self.load_details(refresh=True)
         return self.get_detail("Properties", "Server State").lower() == 'stopped'
 
+    @variable(alias='ui')
```

Comment:
it makes sense to implement this using sentaku

Reply:
It does and we will, but right now middleware has a lot of variant and we'll convert them next year